File: supportx/samples/sample-ios/MockiferSample/Mock/MockiferUtil.swift
import Foundation
import Mockifer

class MockiferUtil {
    static let isMockApp = true
    
    static func pushMock(routeId: String) {
        Mockifer.pushMock(routeId)
    }
    
    static func reset() {
        Mockifer.reset()
    }
}

File: support/samples/sample-android/app/src/mock/java/io/github/marcelbraghetto/mockifer/example/android/MockiferUtil.kt
package io.github.marcelbraghetto.mockifer.example.android

import io.github.marcelbraghetto.mockifer.Mockifer

object MockiferUtil {
    fun pushMock(mockId: String) {
        Mockifer.pushMock(mockId)
    }

    fun reset() {
        Mockifer.reset()
    }
}